packages/editor/src/components/provider/use-block-editor-settings.js

```diff
@@ -130,7 +130,7 @@ function useBlockEditorSettings( settings, hasTemplate ) {
 			reusableBlocks: select( 'core' ).getEntityRecords(
 				'postType',
 				'wp_block',
-				{ per_page: -1 }
+				{ per_page: Platform.select( { web: -1, native: 100 } ) }
```

Comment:
This is a workaround for fixing the issue with [Unbounded queries](https://github.com/wordpress-mobile/gutenberg-mobile/issues/2661) on native.

Reply:
Sorry, I missed this before, I think we should add the mentioned issue as a comment here as well.

Reply:
@ceyhun Good idea!, I've just added a brief comment of the issue directly in the code in https://github.com/WordPress/gutenberg/pull/25265/commits/9f021bb96a6d2fd14c7ae0c26646ef8bd4b965f4.